backend/src/db.ts

```diff
@@ -1,49 +1,50 @@
 import {
 	Folder,
-	Poll,
-	PollOption,
-	PollQuestion,
-	PollUserResponse,
+	Collection,
+	QuestionOption,
+	Question,
+	QuestionUserResponse,
 	User,
 	UserMobileSetting,
 	UserWebSetting,
 } from './models';
 
 const init = async (): Promise<void> => {
-	Folder.hasMany(Poll, { foreignKey: 'folderId' });
-	Poll.belongsTo(Folder, { foreignKey: 'folderId' });
-
-	Poll.hasMany(PollQuestion, { foreignKey: 'pollId' });
-	PollQuestion.belongsTo(Poll, { foreignKey: 'pollId' });
-
-	PollQuestion.hasMany(PollOption, { foreignKey: 'pollQuestionId' });
-	PollOption.belongsTo(PollQuestion, { foreignKey: 'pollQuestionId' });
-
-	PollQuestion.hasMany(PollUserResponse, { foreignKey: 'pollQuestionId' });
-	PollUserResponse.belongsTo(PollQuestion, { foreignKey: 'pollQuestionId' });
-
-	PollOption.hasMany(PollUserResponse, { foreignKey: 'pollUserResponseId' });
-	PollUserResponse.belongsTo(PollOption, { foreignKey: 'pollUserResponseId' });
-
-	User.hasMany(PollUserResponse, { foreignKey: 'userId' });
-	PollUserResponse.belongsTo(User, { foreignKey: 'userId' });
-
-	User.hasOne(UserWebSetting, { foreignKey: 'userId' });
-	UserWebSetting.belongsTo(User, { foreignKey: 'userId' });
-
-	User.hasOne(UserMobileSetting, { foreignKey: 'userId' });
-	UserMobileSetting.belongsTo(User, { foreignKey: 'userId' });
-
 	try {
 		await User.sync({ alter: true });
```

Comment:
You don't need to do this.

The original sequalize object you created in config has all the models within it you just need to run .sync on that object